OpenRA.Mods.D2k/Projectiles/SonicBlast.cs

```diff
@@ -0,0 +1,161 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.GameRules;
+using OpenRA.Graphics;
+using OpenRA.Mods.Common.Graphics;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Projectiles
+{
+	[Desc("Blast projectile that travels in a straight line.")]
+	public class SonicBlastInfo : IProjectileInfo
+	{
+		[Desc("Projectile speed in WDist / tick, two values indicate a randomly picked velocity per blast.")]
+		public readonly WDist[] Speed = { new(128) };
+
+		[Desc("The number of ticks between the blast causing warhead impacts in its area of effect.")]
+		public readonly int DamageInterval = 1;
+
+		[Desc("Equivalent to sequence ZOffset. Controls Z sorting.")]
+		public readonly int ZOffset = 0;
+
+		[Desc("The minimum distance the blast travels.")]
+		public readonly WDist MinDistance = WDist.Zero;
+
+		[Desc("Visual Width of the blast")]
```

Comment:
Can we change this to `[Desc("Width of projectile (used for finding blocking actors).")]` to match Bullet?